sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -233,6 +233,12 @@ export interface DatasetsPendingUploadOptionalParams extends OperationOptions {
 // @public
 export type DatasetType = "uri_file" | "uri_folder";
 
+// @public
+export interface DatasetUploadOptionalOptions {
+    connectionName?: string;
```

Comment:
Why is `connectionName` optional? can we call `uploadFile` or `uploadFolder` without specifying a connection name?

Reply:
Can we name it `DatasetUploadOptions` instead of `DatasetUploadOptionalOptions`? "Options" are implied to be optional..